ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -19,161 +21,68 @@ import StatusQ.Controls 0.1
+                width: implicitWidth
+                text: qsTr("Preview")
             }
         }
 
-        StatusDescriptionListItem {
+        StackLayout {
             Layout.fillWidth: true
-            Layout.leftMargin: Style.current.padding
-            Layout.rightMargin: Style.current.padding
-
-            title: qsTr("ENS username")
-            subTitle: root.profileStore.ensName
-            tooltip.text: qsTr("Copy to clipboard")
-            icon.name: "copy"
-            visible: !!root.profileStore.ensName
-            iconButton.onClicked: {
-                root.profileStore.copyToClipboard(root.profileStore.ensName)
-                tooltip.visible = !tooltip.visible
-            }
-        }
+            currentIndex: editPreviwTabBar.currentIndex
 
-        StatusDescriptionListItem {
-            Layout.fillWidth: true
-            Layout.leftMargin: Style.current.padding
-            Layout.rightMargin: Style.current.padding
-
-            title: qsTr("Chat key")
-            subTitle: Utils.getCompressedPk(root.profileStore.pubkey)
-            subTitleComponent.elide: Text.ElideMiddle
-            subTitleComponent.width: 320
-            subTitleComponent.font.family: Theme.palette.monoFont.name
-            tooltip.text: qsTr("Copy to clipboard")
-            icon.name: "copy"
-            iconButton.onClicked: {
-                root.profileStore.copyToClipboard(subTitle)
-                tooltip.visible = !tooltip.visible
+            MyProfileSettingsView {
+                id: settingsView
+                Layout.fillWidth: true
             }
-        }
 
-        StatusDescriptionListItem {
-            Layout.fillWidth: true
-            Layout.leftMargin: Style.current.padding
-            Layout.rightMargin: Style.current.padding
-
-            title: qsTr("Share Profile URL")
-            subTitle: `${Constants.userLinkPrefix}${root.profileStore.ensName !== "" ? root.profileStore.ensName : (root.profileStore.pubkey.substring(0, 5) + "..." + root.profileStore.pubkey.substring(root.profileStore.pubkey.length - 5))}`
-            tooltip.text: qsTr("Copy to clipboard")
-            icon.name: "copy"
-            iconButton.onClicked: {
-                root.profileStore.copyToClipboard(Constants.userLinkPrefix + (root.profileStore.ensName !== "" ? root.profileStore.ensName : root.profileStore.pubkey))
-                tooltip.visible = !tooltip.visible
+            MyProfilePreview {
+                id: profilePreview
+                Layout.fillWidth: true
+                profileStore: root.profileStore
             }
         }
 
-        Component {
-            id: displayNamePopupComponent
-            DisplayNamePopup {
-                profileStore: root.profileStore
-                anchors.centerIn: Overlay.overlay
-                onClosed: { destroy() }
-            }
+        ChangePasswordModal {
+            id: changePasswordModal
+            privacyStore: root.privacyStore
```

Comment:
@igor-sirotin you're referring to the `privacyStore` of `MyProfileView`, but you don't have it defined there?